components/menu/Menu.js

```diff
@@ -5,7 +5,7 @@ import classnames from 'classnames';
 import { themr } from 'react-css-themr';
```

Comment:
this commit is a bit confusing because it uses code that isn't introduced until next commit